src/display/annotation_layer.js

```diff
@@ -2048,6 +2057,7 @@ class FreeTextAnnotationElement extends AnnotationElement {
     );
     super(parameters, { isRenderable, ignoreBorder: true });
     this.textContent = parameters.data.textContent;
+    this.annotationEditorType = AnnotationEditorType.FREETEXT;
```

Comment:
If this is actually necessary, why is there not a similar entry for `Ink`-annotations?